index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Homepage</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <div class="wrapper">
+        <div class="logo">
+          <img src="https://i.postimg.cc/mg4rWBmv/logo.png" alt="logo" />
+        </div>
+        <ul class="nav-area">
+          <li><a href="#">Home</a></li>
```

Comment:
remove target and use nav tabs on all pages, so that you don't have to go back to home page for navigation